src/index.html

```diff
@@ -9,7 +9,7 @@
   <script src="https://cdn.jsdelivr.net/npm/zone.js@0.11.4/bundles/zone.umd.min.js"></script>
   <script src="https://cdn.jsdelivr.net/npm/@laserfiche/lf-ui-components@15/cdn/lf-ui-components.js" defer></script>
   <link href="https://cdn.jsdelivr.net/npm/@laserfiche/lf-ui-components@15/cdn/lf-laserfiche-lite.css" rel="stylesheet" />
-  <link href="https://cdn.jsdelivr.net/npm/@laserfiche/lf-ui-components@15/cdn/indigo-pink.css" rel="stylesheet" />
+  <!-- <link href="https://cdn.jsdelivr.net/npm/@laserfiche/lf-ui-components@14/cdn/indigo-pink.css" rel="stylesheet" /> -->
```

Comment:
because we get the scripts from the cdn here you could go either way, I would go with the change you made, but remove line 12 in html